src/platform-includes/configuration/capture-console/javascript.mdx

```diff
@@ -14,27 +14,47 @@ Sentry.init({
 });
 ```
 
-```javascript {tabTitle: CDN}
+```html {tabTitle: Loader}
+<!-- In addition to the Loader: -->
```

Comment:
Can we not include the loader URL in here?

Reply:
I wasn't 100% sure if we want to include this in every one of this 🤔 As it kind of drowns out the "relevant" change a bit, IMHO. But not sure, happy to put it in if we think that makes more sense!

Reply:
I would like to see it to make it extra clear what is going on - also signifies order to users.